lib/rules/no-restricted-modules.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * @author Christian Schulz
+ * See LICENSE file in root directory for full license.
+ */
+"use strict"
+
+const ignore = require("ignore")
```

Comment:
Looks like this dependency is already in this package.json 🎉

Reply:
Yes, this plugin uses `ignore` package to parse `.npmignore` files and a rule checks if ignored files are imported from published files.